docs/slides/multi-cloud/adp/index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>HashiCorp Field Workshops</title>
```

Comment:
You might want to give a more specific header title.

Reply:
Changed to   "<title>HashiCorp Vault - Advanced Data protection</title>"